src/specs/av1_isobmff/av1_isobmff.cpp

```diff
@@ -465,6 +466,7 @@ const SpecDesc specAv1ISOBMFF = {
       } },
     { "Section 2.1\n"
       "It SHALL contain at least one track using an AV1SampleEntry",
```

Comment:
"It SHALL contain at least one track using an AV1SampleEntry, possibly transformed by encryption as specified in § 4 Common Encryption",
  "assert-bd1c6212",